tests/test_flatland_envs_rail_env_shortest_paths.py

```diff
@@ -112,13 +113,21 @@ def test_get_shortest_paths():
             Waypoint(position=(2, 4), direction=3),
             Waypoint(position=(2, 3), direction=3),
             Waypoint(position=(2, 2), direction=3),
-            Waypoint(position=(2, 1), direction=3)]
+
+            # Change a point to test the assertion works :)
+            Waypoint(position=(2, 1), direction=3)
+            #Waypoint(position=(2, 2), direction=3)
+            ]
     }
 
-    for agent_handle in expected:
-        assert np.array_equal(actual[agent_handle], expected[agent_handle]), \
-            "[{}] actual={},expected={}".format(agent_handle, actual[agent_handle], expected[agent_handle])
+    for iA, lWP in expected.items():
+        _compare_paths(iA, actual[iA], lWP)
 
+def _compare_paths(iAgent:int, actual:List[Waypoint], expected:List[Waypoint]):
```

Comment:
maybe name this function `_assert_path` to hint that it carries out assertions?

Reply:
sounds entirely reasonable.  Maybe _assert_paths_equal.

Reply:
I changed it to _assert_paths_eequal